tests/test_smartcar.py

```diff
@@ -189,7 +189,56 @@ def test_get_auth_url_vehicle_info_dictionary(self):
         })
 
         expected = smartcar.const.CONNECT_URL + '/oauth/authorize?' + query
-        
+
+        expected_params = parse_qs(expected)
+        actual_params = parse_qs(actual)
+
+        assertDeepEquals(self, expected_params, actual_params)
+
+    def test_get_auth_url_single_select_bool(self):
```

Comment:
Add test for without single select?

Reply:
:+1:

Reply:
actually didn't add this one because it is implicitly tested in our default getAuthUrl